src/Administration/Resources/app/administration/package.json

```diff
@@ -49,7 +49,7 @@
   ],
   "dependencies": {
     "@shopware-ag/meteor-admin-sdk": "6.0.0",
-    "@shopware-ag/meteor-component-library": "^4.12.1",
+    "@shopware-ag/meteor-component-library": "4.12.2",
```

Comment:
Have you double checked the unrelated changes and tested these as well?
Looking through
 https://github.com/shopware/meteor/compare/%40shopware-ag/meteor-component-library%404.12.1...%40shopware-ag/meteor-component-library%404.12.2

It looks like there are also slight adjustments to components like `mt-select-selection-list.vue`, `mt-help-text.vue`, `mt-select.vue`, `mt-tooltip.vue` as well as a bunch of dependency updates.

cc: @jc-sw-git

Reply:
I did. They were just backwards compatible changes